lib/Config/FetcherConfig.php

```diff
@@ -125,9 +125,16 @@ public function __construct(
             $url = $url->withUserInfo($auth[0], $auth[1]);
         }
 
+        // prevent broken scheme '//'
+        $scheme = $url->getScheme();
```

Comment:
`//` just means use the same protocol as you're already using so I don't think we can just assume the protocol based on that. I think it would be better to error without a protocol and add frontend validation.

Reply:
This is for the proxy config, it's not related to feeds. 

The proxy config is set via the nextcloud config file.

If the proxy config has no schema guzzel adds // which causes errors during fetching a feed see linked issue

Reply:
Well if user puts some.domain:9010 as proxy config, then we do not connect at all since the URi lib will add // in front so `//some.domain:9010`

Instead we have an error, I'm not sure what the old implementation did maybe leave the domain without schema or add http who knows.